docker-compose.yml

```
@@ -10,7 +10,11 @@ services:
       - postgres
     volumes:
       - ./script:/app
+    environment:
+      - MAPBOX_ACCESS_TOKEN=sk.eyJ1Ijoic2hhbGluaTEyOTciLCJhIjoiY2wwZ2lsNnlkMTM4bDNjazZvc3VzNHc1OCJ9.LdWiqSVbOpUkWJYEp6QY2A
```

don't put access token in code directly. put it in your `.bashrc` or `.zshrc` and pass via environment variables
```suggestion
      - MAPBOX_ACCESS_TOKEN=${MAPBOX_ACCESS_TOKEN}
```

and then your `.bashrc` or `.zshrc` file can look like 

```
export MAPBOX_ACCESS_TOKEN=sk.eyJ1Ijoic2hhbGluaTEyOTciLCJhIjoiY2wwZ2lsNnlkMTM4bDNjazZvc3VzNHc1OCJ9.LdWiqSVbOpUkWJYEp6QY2A

```